indico/modules/rb/operations/bookings.py

```diff
@@ -193,14 +203,16 @@ def get_room_calendar(start_date, end_date, room_ids, include_inactive=False, **
                                  include_inactive=include_inactive))
     query = query.order_by(db.func.indico.natsort(Room.full_name))
     rooms = (Room.query
-             .filter(~Room.is_deleted, Room.id.in_(room_ids) if room_ids else True)
+             .filter(~Room.is_deleted, Room.id.in_(room_ids))
              .options(joinedload('location'))
              .order_by(db.func.indico.natsort(Room.full_name))
              .all())
 
+    bookings = query.all()
+    rooms.extend(set(b.reservation.room for b in bookings if b.reservation.room not in rooms))
```

Comment:
what's the reason for the `not in rooms` check here? Just make `rooms` a set :) Sorting is broken here anyway after adding stuff to the list, so you need to re-sort anyway (and you can remove the `order_by` then)

Reply:
I think it was due to some assumption I made initially which turned out to be incorrect. I think it can be removed altogether